cli/BUILD

```diff
@@ -153,3 +153,13 @@ js_binary(
     # required to make npm to be available in PATH
     include_npm = True,
 )
+
+js_binary(
+    name = "dsl_bin",
+    data = [
+        "package.json",
+        ":player-cli-bundle",
+    ] + dependencies,
+    entry_point = "bin/run",
+    visibility = ["//visibility:public"],
+)
```

Comment:
What do we need this for?

Reply:
It's in use [here](https://github.com/player-ui/tools/blob/cdb4a40ad7f0b8c894ca0aff2ebadad6f0e8cdfa/devtools/plugins/desktop/basic/BUILD#L48), and it'll be leveraged by any other dev tools plugin we add, as tool for compiling the DSL.

Reply:
Can we make it a maco in rules_player that takes a param for the cli tag to use that way other repos can use the one thats pulled in as a dependency and this repo can use the local one.

Reply:
The DSL implementation in the rules project is still ongoing. Can we defer the changes on whether or not to have the macro to an issue there? That extrapolates the scope of this PR. 
@adierkens , could you consider that while working on the new player rules/macros?